### PR TITLE
Alternate light and dark rows on the teacher homepage sections table

### PR DESCRIPTION
**Pull request by Erin007**
The sections table on the teacher homepage now has alternating light and dark rows and makes links less bold. 

BEFORE:
sections table before

AFTER:
screen shot 2017-06-19 at 8 57 16 pm

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

**Comment by tanyaparker** [comment]
Sorry, meant to take this off your list since Brent was converting the sections table to React and we were going to do a style touch up later. But since you already did it, we need to fine tune the colors a little. :)

The column headers should be styled like the grey (I think #f6f6f6?) default buttons. Same grey and also that little white bar at the top that makes it pop out/not flat.

Then the rows should alternate between white and #fcfcfc. It looks like in your after screenshot it has 3 colors?

image

And redlines if this helps:

image

**Comment by tanyaparker** [comment]
Oh! And would it be possible to make the links in the table `"Gotham 5r", sans-serif` instead of 7r? :):)

**Comment by tanyaparker** [comment]
Alternating row colors + less bold links are the changes I'm recommending. Actually, skip making the column header 3D. Not needed and we may not keep the 3D style with the redesign. So looks more like this (this mock is from editing your page as opposed to Mike's mockup):

image

**Comment by Bjvanminnen** [comment]
I had recommended Erin style this after the table we use for our progress rows (i.e. in terms of row colors). If these arent the right colors, should we be changing that table too?

**Comment by tanyaparker** [comment]
@poorvasingal fyi

I believe the progress rows (I'm assuming you meant the unit overview page?) are different because they have a lot of white bubbles. So the white on white wouldn't work well. Poorva, would you rather these two tables be consistent or are you ok with the differing styles? If they were to be the same, then I would say go with the progress row colors.

File: apps/src/templates/studioHomepages/SectionsTable.jsx [hunk 1]
```diff
@@ -117,7 +122,13 @@ const SectionsTable = React.createClass({
         </thead>
         <tbody>
           {sections.map((section, index) =>
-            <tr style={styles.row} key={index}>
+            <tr
+              style={{
+                ...(!index%2 && styles.lightRow),
+                ...(index%2 && styles.darkRow), ...styles.row
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/SectionsTable.jsx]: nit: styles.row should be on new line

Realize I said to do it the way you have it, but I also wonder if the following would be marginally easier to understand:
```
...(index % 2 === 0 ? styles.lightRow : styles.darkRow)
...styles.row
```
thoughts?

**Pull request merged**